mongodbatlas/resource_mongodbatlas_org_invitation_test.go

```diff
@@ -0,0 +1,198 @@
+					resource.TestCheckResourceAttr(resourceName, "username", name),
+					resource.TestCheckResourceAttr(resourceName, "roles.#", "1"),
+				),
+			},
+			{
+				Config: testAccMongoDBAtlasOrgInvitationConfig(orgID, name, updateRoles),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckMongoDBAtlasOrgInvitationExists(resourceName, &invitation),
+					testAccCheckMongoDBAtlasOrgInvitationUsernameAttribute(&invitation, name),
+					testAccCheckMongoDBAtlasOrgInvitationRoleAttribute(&invitation, updateRoles),
+					resource.TestCheckResourceAttrSet(resourceName, "username"),
+					resource.TestCheckResourceAttrSet(resourceName, "invitation_id"),
+					resource.TestCheckResourceAttr(resourceName, "name", name),
+					resource.TestCheckResourceAttr(resourceName, "roles.#", "2"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccResourceMongoDBAtlasOrgInvitation_importBasic(t *testing.T) {
+	var (
+		resourceName = "mongodbatlas_invitations.test"
+		orgID        = os.Getenv("MONGODB_ATLAS_ORG_ID")
+		name         = fmt.Sprintf("test-acc-%s", acctest.RandString(10))
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: testAccProviderFactories,
+		CheckDestroy:      testAccCheckMongoDBAtlasOrgInvitationDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccMongoDBAtlasOrgInvitationConfig(orgID, name, []string{"mongodbatlas.testing@gmail.com"}),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet(resourceName, "org_id"),
+					resource.TestCheckResourceAttrSet(resourceName, "name"),
+					resource.TestCheckResourceAttrSet(resourceName, "usernames.#"),
+					resource.TestCheckResourceAttrSet(resourceName, "invitation_id"),
+
+					resource.TestCheckResourceAttr(resourceName, "org_id", orgID),
+					resource.TestCheckResourceAttr(resourceName, "name", name),
+					resource.TestCheckResourceAttr(resourceName, "usernames.#", "1"),
+				),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportStateIdFunc: testAccCheckMongoDBAtlasOrgInvitationStateIDFunc(resourceName),
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func testAccCheckMongoDBAtlasOrgInvitationExists(resourceName string, invitation *matlas.Invitation) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		conn := testAccProvider.Meta().(*MongoDBClient).Atlas
+
+		rs, ok := s.RootModule().Resources[resourceName]
+		if !ok {
+			return fmt.Errorf("not found: %s", resourceName)
+		}
+
+		ids := decodeStateID(rs.Primary.ID)
+
+		orgID := ids["org_id"]
+		username := ids["username"]
+		invitationID := ids["invitation_id"]
+
+		if orgID == "" && username == "" && invitationID == "" {
+			return fmt.Errorf("no ID is set")
+		}
+
+		log.Printf("[DEBUG] orgID: %s", orgID)
+		log.Printf("[DEBUG] username: %s", username)
+		log.Printf("[DEBUG] invitationID: %s", invitationID)
```

Comment:
[nit] pass `t *testing.T` as a parameter to this function and do 
```suggestion
		t.Logf("orgID: %s", orgID)
		t.Logf("username: %s", username)
		t.Logf("invitationID: %s", invitationID)
```

Also not sure if we have that liner in this repo but since this is a function helper be sure to call `t.Helper()` as the first line of the function after adding `t` as a param